prepareLocalDevelopment.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+PROJECT_DIRECTORY=$1
+LOCAL_PURCHASES_HYBRID_COMMON_DIRECTORY=$2
+SETTINGS_GRADLE_FILE=$PROJECT_DIRECTORY/settings.gradle
+
+echo "" >> $PROJECT_DIRECTORY/settings.gradle
```

Comment:
This is so the next line is guaranteed to be in a new line. Otherwise, it would append to the contents of the latest line instead, and since there's no empty line at the end of the file, it just appends it like this:

```
include ':launcher', ':unityLibrary'includeBuild('/Users/cesardelavega/Documents/Development/repos/purchases-hybrid-common/android')
```

Reply:
perhaps include that as a comment?